models/sudoswap/ethereum/sudoswap_ethereum_events.sql

```diff
@@ -346,7 +346,7 @@ WITH
             , aggregator_address
             , aggregator_name
             , platform_fee_amount/number_of_items as platform_fee_amount
-            , platform_fee_amount_raw/number_of_items as platform_fee_amount_raw
+            , cast(platform_fee_amount_raw/number_of_items as double) as platform_fee_amount_raw
```

Comment:
If we need to do it here don't we also need to cast the divisions below?

Reply:
No I checked and the other ones are actually in the right format